### ISSUE TITLE
VPAT 2.4.6, Labels and Instructions for Forms

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#18]
Screengrab/videos (insert file names here)
1, Search Box description.png
2, Lookup on 2nd screen.png
3, Lookup on 1st page.png

Link
https://permit-finder.digital-staging.boston.gov

Environment
(Device > OS > Browser)
Computer > macOS 10.14.6 > Safari 12.1.2

Brief Description
Instructions are not sufficient for assistive technology.

Steps to reproduce
1. See “1, Search Box description.png”.  The Permit Number field is labeled as “Search Box”.  Additional information would be helpful, such as “Enter a permit number”
2. See “2, Lookup on 2nd screen.png”  The Lookup Button on the detail screen is labelled differently than on the landing page.

Desired Result
Sufficient instructions in the label to define the functionality.

Actual Result
Additional instructions are available on screen, but are not associated with the label.

![3, Lookup on 1st page](https://user-images.githubusercontent.com/33845414/62466877-67ef7300-b760-11e9-9d5b-7ffe9b35f25d.png)
![1, Search Box description](https://user-images.githubusercontent.com/33845414/62466879-67ef7300-b760-11e9-8011-c0f51b3a1fd7.png)
![2, Lookup on 2nd screen](https://user-images.githubusercontent.com/33845414/62466880-68880980-b760-11e9-8522-6af8139ecc27.png)
